SummerSchool/RGSForumThreadTableViewCell.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  RGSForumThreadTableViewCell.swift
+//  SummerSchool
+//
+//  Created by Charles Randolph on 12/13/17.
+//  Copyright © 2017 RUG. All rights reserved.
+//
+
+import UIKit
+
+class RGSForumThreadTableViewCell: UITableViewCell {
+    
+    // MARK: - Variables & Constants
+    
+    /// The Author's profile image.
+    var authorImage: UIImage! {
+        didSet (oldAuthorImage) {
+            if (authorImage != nil) {
+                authorImageView.image = authorImage;
+            }
+        }
+    }
+    
+    // The posting date of the thread.
+    var date: Date! {
+        didSet (oldDate) {
+            if (date != nil) {
+                let time = DateManager.sharedInstance.dateToISOString(date, format: .hoursAndMinutesFormat)
+                dateLabel.text = time! + ", " + DateManager.sharedInstance.dateToISOString(date, format: .generalPresentationDateFormat)!
+            }
+        }
+    }
+    
+    // The identify of the author.
+    var author: String! {
+        didSet (oldAuthor) {
+            if (author != nil) {
+                authorLabel.text = author;
+            }
+        }
+    }
+    
+    // The title of the forum thread.
+    var title: String! {
```

Comment:
there should be somewhere a comment to explain this pattern of 
`didSet(oldStuff) { label.text = stuff }`
what is oldStuff?

Reply:
Yes, I will add them. didSet (old<varname>) is called once a variable has changed. The old value  may be accessed and you can decide if you want to keep or discard it.

Reply:
In retrospect, it should have been obvious :)